model/patch/patch.go

```diff
@@ -628,6 +640,25 @@ func (p *Patch) IsChild() bool {
 	return p.Triggers.ParentPatch != ""
 }
 
+func (p *Patch) SetParametersFromParent() error {
+	parentPatchId := p.Triggers.ParentPatch
+	parentPatch, err := FindOneId(parentPatchId)
+	if err != nil {
+		return errors.Wrap(err, "can't get parent patch")
+	}
+	if parentPatch == nil {
+		return errors.Wrap(err, fmt.Sprintf("parent patch '%s' does not exist", parentPatchId))
+	}
+	if downstreamParams := parentPatch.Triggers.DownstreamParameters; len(downstreamParams) > 0 {
+		p.Parameters = downstreamParams
```

Comment:
we should be able to skip setting `p.Parameters` here because the call `p.SetParameters` will do it.